server/hot-reloader.js

```diff
@@ -1,6 +1,7 @@
 import { join, relative, sep } from 'path'
 import webpackDevMiddleware from 'webpack-dev-middleware'
 import webpackHotMiddleware from 'webpack-hot-middleware'
+import isWindowsBash from 'is-windows-bash'
```

Comment:
We use absolute versions, so use that. I mean versions like `0.8.1` not `^0.8.1`